R/helpers.R

```diff
@@ -42,6 +42,31 @@ Validate <- setRefClass(
   )
 )
 
+#' Combining validation results
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' A simple helper function that combines outputs from S4 object validation
+#' functions into one consistent object. Namely, it combines the validation
+#' `character` messages supplied to the input of this function into one `character`
+#' vector, or simply returns `TRUE` if there were no such messages.
+#'
+#' @param ... outputs from S4 object validation functions.
+#'
+#' @return `character` vector if there was at least object of type `character`
+#'   in the input. Otherwise, `TRUE`.
+#'
+#' @export
+h_combine_validation_results <- function(...) {
```

Comment:
I thought we had already a helper for this? or not?

Reply:
Nope, this is a new thing.